ludwig/combiners/combiners.py

```diff
@@ -810,7 +811,7 @@ def forward(
             hidden = torch.permute(hidden, (1, 0, 2))  # bs, num_eo, h
 
             if self.embed_input_feature_name:
-                i_f_names_idcs = torch.reshape(torch.arange(0, len(embeddable_encoder_outputs)), [-1, 1])
+                i_f_names_idcs = torch.reshape(torch.arange(0, len(embeddable_encoder_outputs)).to(DEVICE), [-1, 1])
```

Comment:
@anmshkmr the issue here seems to be that these tensors are being constructed at runtime. Instead, we should aim to construct them as properties of the module during initialization, that way they will be placed on the correct device when we call `model.cuda()`.